api.md

```diff
@@ -3139,33 +3139,31 @@ SELECT get_telemetry_report(always_display_report := true);
 
 ---
 
-## hypertable_approximate_row_count() [](hypertable_approximate_row_count)
+## approximate_row_count() [](approximate_row_count)
 
-Get approximate row count for hypertable(s) based on catalog estimates.
+Get approximate row count for hypertable, distributed hypertable or regular PostgreSQL table based on catalog estimates.
```

Comment:
Super minor, but oxford commas in the docs (I already had one other suggestion, or otherwise I wouldn't be making this =)

```suggestion
Get approximate row count for hypertable, distributed hypertable, or regular PostgreSQL table based on catalog estimates.
```